src/CarrierLogo/__tests__/index.test.js

```diff
@@ -2,7 +2,8 @@
 import * as React from "react";
 import { mount } from "enzyme";
```

Comment:
Combination of `mount` and `ThemeProvider` is crazy. It generates a very large snapshot. I'll rewrite the test.

Reply:
okay, thanks